assets/src/scss/pages/post/_comments-form.scss

```diff
@@ -41,14 +41,15 @@
 #gdpr-comments-compliance {
   font-family: $roboto;
 
```

Comment:
We could add a `margin-bottom: 8px;` here to make it evenly spaced between the form elements on top and the submit button.